lib/blockchain.go

```diff
@@ -5968,15 +5944,7 @@ func (bc *Blockchain) CreateAtomicTxnsWrapper(
 
 		// Use EstimateFee to set the fee INCLUDING the wrapper. Note that this fee should generally be a bit
 		// higher than the totalFee computed above because the atomic wrapper adds overhead.
-		newFeeEstimate, err := mempool.EstimateFee(
-			atomicTxn,
-			0,
-			// TODO: Make these flags or GlobalParams
-			bc.params.DefaultMempoolCongestionFactorBasisPoints,
-			bc.params.DefaultMempoolPriorityPercentileBasisPoints,
-			bc.params.DefaultMempoolPastBlocksCongestionFactorBasisPoints,
-			bc.params.DefaultMempoolPastBlocksPriorityPercentileBasisPoints,
-			maxBlockSizeBytes)
+		newFeeEstimate, err := mempool.EstimateFee(atomicTxn, 0, maxBlockSizeBytes)
```

Comment:
Should we pass in `MinFeeRateNanosPerKB` here instead of 0? I think I missed this call site in my earlier PR.